android/app/src/main/kotlin/com/fariszr/dualmate/MainActivity.kt

```
@@ -161,4 +165,37 @@ class MainActivity : FlutterActivity() {
+            Intent(
+                Settings.ACTION_APPLICATION_DETAILS_SETTINGS,
+                Uri.parse("package:$packageName")
+            )
+        )
+    }
+
+    private fun tryOpenSettings(settingsIntent: Intent): Boolean {
+        return try {
+            startActivity(settingsIntent)
+            true
+        } catch (error: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)